### PR TITLE
[vscode-extension] fix vscode terminal PATH 

### PR DESCRIPTION
**Pull request by mohsenari**
## Summary
Fixes the issue where terminal's $PATH env variable wouldn't get updated in integrated terminal.

## How was it tested?
buckle up, the steps are long:

- `cd vscode-extension`
- `vsce package` (install vsce via `npm i -g vsce`)
- `code --install-extension devbox-0.0.7.vsix`
- setup a devbox.json in a sample directory with a couple of packages (e.g., `hello`, `go@1.18`)
- open the sample directory in vscode
- cmd+shift+p and run `Devbox: Reopen in Devbox shell environment`
- after vscode reopens, echo $PATH in terminal

### CODE REVIEW

**Comment by Lagoja** [comment]
It looks like it's working!!! I've been messing with my settings a bit so it would be good to get others on the team to test this, but it's worked on Go, Elixir, and Python projects so far.

File: vscode-extension/src/devbox.ts [hunk 1]
```diff
@@ -21,16 +22,17 @@ export async function devboxReopen() {
...
-                            cwd: workingDir.path
-                        });
-                    }
+                    progress.report({ message: 'Installing devbox packages...', increment: 25 });
+                    await setupDotDevbox(workingDir, dotdevbox);
+
+                    // setup required vscode settings
+                    progress.report({ message: 'Updating configurations...', increment: 50 });
+                    updateVSCodeConf();
+                    setTimeout(() => { }, 1000);
```
Comment (LucilleH) [inline vscode-extension/src/devbox.ts:31]: Why do we need the `setTimeout`?

File: vscode-extension/src/devbox.ts [hunk 2]
```diff
@@ -21,16 +22,17 @@ export async function devboxReopen() {
...
+
+                    // Calling CLI to compute devbox env
+                    progress.report({ message: 'Calling Devbox to setup environment...', increment: 80 });
+                    // callDevboxIntegrate(workingDir, progress.report, resolve, reject);
```
Comment (LucilleH) [inline vscode-extension/src/devbox.ts:35]: ✂️

**Review by LucilleH — APPROVED** [review]
🎉

File: vscode-extension/src/devbox.ts (hunk 1 above)
Comment (mohsenari) [reply vscode-extension/src/devbox.ts:31]: ugh, that was from debugging, I'll remove it.

**Pull request merged**